reference/index.rst

```diff
@@ -0,0 +1,20 @@
+Reference
+=========
+
+* :doc:`/reference/annotation/index`
+
+* Configuration Formats
+
+    * :doc:`/reference/format/acl`
+    * :doc:`/reference/format/dashboard`
+    * :doc:`/reference/format/datagrid`
+    * :doc:`/reference/format/data_source`
+    * :doc:`/reference/format/entity_config`
+    * :doc:`/reference/format/navigation`
+    * :doc:`/reference/format/placeholders`
+    * :doc:`/reference/format/requirejs`
+
+.. toctree::
+    :maxdepth: 1
+
+    annotation/index
```

Comment:
Sorry @REDucka, I missed your comment here. I now made the change you requested. However, shouldn't have this worked before as we already had references to those two files in the `index.rst` file from the root directory?